### DISCUSSION TITLE
Is ThmViewer.exe still available for V4 or V5?

### DISCUSSION THREAD

**Discussion by eykh** [wixtoolset/issues#8724] [Questions] [answered]
I only found it in V3.x nuget packgages and that version does not seem to work with current (V4) themes, because it just shows a black window.

I guess I must have missed something obvious.

**Comment by robmen** [comment, marked as answer]
It's in the WixAdditionalTools bundle and is available with each [release of the WiX Toolset](https://github.com/wixtoolset/wix/releases).

**Reply by eykh** [reply]
Yessss, thank you!
